src/docker/composebuilder/service_test.go

```diff
@@ -40,7 +40,8 @@ var _ = Describe("ComposeBuilder", func() {
 			It("should include the docker config for the service itself", func() {
 				dockerConfig, exists := dockerConfigs["mongo"]
 				Expect(exists).To(Equal(true))
-				Expect(dockerConfig.DependsOn).To(ConsistOf([]string{"exocom0.26.1", "mongo3.4.0"}))
+				Expect(dockerConfig.DependsOn[0]).To(Equal("exocom0.26.1"))
+				Expect(dockerConfig.DependsOn[1]).To(Equal("mongo3.4.0"))
```

Comment:
We can keep this as a single line with `Equal(...)` instead of `ConsistOf(...)` http://onsi.github.io/gomega/#equalexpected-interface